app/views/authorised_to_view/authorised_to_view_search.scala.html

```diff
@@ -32,7 +32,12 @@
         button: components.button
 )
 
-@(form: Form[_], url: Option[String], date: String, fileExists: Boolean)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(
+form: Form[_],
```

Comment:
Whats our thinking for have each param with 0 indentation on the left edge? I can see we're doing a few times here.

Reply:
The only idea was to keep parameters on the separate lines in order to make these easily readable as it is very difficult to read all the parameters are on a single line. We have done the same for this constructor for the views. Indentation has been added now. Have a look and let me know if that's okay

Reply:
Reviewer is happy with the change. Hence resolving.